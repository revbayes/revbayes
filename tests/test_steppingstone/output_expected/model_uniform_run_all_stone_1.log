Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	gamma_rates[1]	gamma_rates[2]	gamma_rates[3]	gamma_rates[4]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-21295.8	-21274.82	-20.98634	0.7425676	0.131885	0.140754	0.199467	0.121813	0.126913	0.200598	0.370288	0.118727	0.219831	0.108953	0.343492	0.238036	0.167588	0.213465	0.196561	0.119627	0.141093	0.163938	0.14116	0.300376	0.362293	0.181619	0.395866	0.129709	0.0625595	0.000660948	0.126722	0.0643629	0.0557914	0.249375	0.287927	0.0478006	0.0575662	0.00661745	0.0338467	0.0866382	0.0413013	0.0594353	0.0143173	0.0263351	0.228671	0.176727	0.262228	0.0266069	0.371647	0.0496825	0.034346	0.476093	0.0416247	0.0832941	0.382941	0.939676	2.59409	0.384017	0.321174	0.0778758	0.216933	0.3767456	6.722933
10	-21303.1	-21287.46	-15.63601	0.7521828	0.121584	0.122653	0.136734	0.092678	0.149556	0.261148	0.246111	0.118947	0.159163	0.116615	0.225457	0.155204	0.157228	0.245357	0.186112	0.125146	0.130302	0.186371	0.191158	0.225921	0.301553	0.178451	0.395866	0.0909928	0.0420447	0.000539831	0.131301	0.084885	0.0422418	0.209508	0.317871	0.0553328	0.0781383	0.0067273	0.0354043	0.0860687	0.0412516	0.0427343	0.0442967	0.0132368	0.191213	0.166102	0.28108	0.0274941	0.38404	0.0561598	0.0191389	0.470939	0.0422286	0.0853515	0.387146	0.942771	2.58473	0.378362	0.316444	0.0767289	0.228465	0.3972708	6.190283
