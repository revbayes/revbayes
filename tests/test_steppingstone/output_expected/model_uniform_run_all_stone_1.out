state	power	likelihood
4	1	-21258.8
5	1	-21257
6	1	-21259.2
7	1	-21256.5
8	1	-21256.7
9	1	-21258
10	1	-21263.2
